examples/hello-world/index.js

```diff
@@ -2,12 +2,12 @@ var express = require('../../');
 
 var app = express();
 
-app.get('/', function(req, res){
+app.get('/', function(req, res) {
   res.send('Hello World');
 });
 
 /* istanbul ignore next */
-if (!module.parent) {
+if (module.children) {
```

Comment:
I'm not sure this change is correct looking at the docs (even though it happens to work for our tests). The current conditional tests to see if this file was required by another module or not (desired test), but this change instead checks if any module require by this module was the first time is was required or not. That seems like it would be a side effecr, as this only works if express was required prior to requiring thispdule, which is not the purpose of the test here.

Reply:
This is a bit out of my knowledge now. I originally didn't intend on changing that line, however, I did because the docs showed that `module.parent` was deprecated. I'll let you and Aravind decide the best solution for this.

Reply:
Gotcha. If that is the case, it seems like this PR should then be closed and one of us will open a new one with the appropriate change.

Reply:
Sounds good. If you'd like me to do that, I can. Otherwise have a nice day!